src/06-excitation-transfer-revisited.md

```diff
@@ -0,0 +1,696 @@
+    nmm_list = [(n,m1,m2) for (n,m1,m2) in product(possible_ns, possible_ms, possible_ms)]
+    
+    # only do parity extraction if a valid parity is being used
+    if (parity==1) | (parity==-1):
+        p           = np.where(P.diag()==parity)[0]
+    
+        two_state_1     = two_state_1.extract_states(p)
+        two_state_2     = two_state_2.extract_states(p)
+        bosons          = bosons.extract_states(p)
+        number          = number.extract_states(p)
+        interaction_1   = interaction_1.extract_states(p)
+        interaction_2   = interaction_2.extract_states(p)
+        P               = P.extract_states(p)
+        J2              = J2.extract_states(p)
+        nmm_list        = [nmm_list[i] for i in p]
+
+    return two_state_1, two_state_2, bosons, interaction_1, interaction_2, number, nmm_list
+```
+
+The most obvious features we can play with here are:
+- $\Delta E_1$ & $\Delta E_2$
+- $U_1$ & $U_2$
+
+Let's explore
+
+
+## 6.2 - Matched TLS
+
+
+The simplest way to explore excitation transfer is to "match up" the TLS $\Delta E_1 = \Delta E_2 = \Delta E$ and $U_1 = U_1 = U$.
+
+$$H =  \frac{\Delta E}{2} (\sigma_{z1} + \sigma_{z2}) + \hbar\omega\left(a^{\dagger}a +\frac{1}{2}\right) + U\left( a^{\dagger} + a \right)(\sigma_{x1} + \sigma_{x2})$$
+
+
+```python
+H_latex = "$H = \Delta E /2 (\sigma_{z1} + \sigma_{z2}) + \hbar\omega(a^{{\dagger}}a +1/2) + U( a^{{\dagger}} + a )(\sigma_{x1} +\sigma_{x2} )$ "
+```
+
+ In other words, we have identical TLS that interact with the boson field in the same way. This is the same set up as in the last tutorial, only now we're going to see how the excitation transfer time depends on $\Delta E$ and $U$.
+
+
+
+
+Let's set the boson energy to be $\omega = 1$ as we've done in previous tutorials.
+
+```python
+omega = 1
+```
+
+We'll increase the coupling from last time to $U = 0.1$ to make our explorations easier from a computational perspective.
+
+```python
+U = 0.1
+```
+
+And we'll focus on even parity
+
+```python
+parity = 1
+```
+
+Let's start by looking at $\Delta E$ and then $U$.
+
+
+### $\Delta E$ scaling
+
+<!-- #region -->
+
+
+In general, it's not possible to simply vary $\Delta E$ continuously across large range and look at how long it takes for e.g. $|1,+, - \rangle$ to transform into $|1,-, + \rangle$. The reason is that "excitation transfer" cannot be totally separated from  what we might call `energy exchange`. 
+
+Energy exchange occurs when $\Delta E = n\omega$. That's when spontaneous emission and down convertsion can occur. Close to those "resonances", energy exchange dominates over excitation transfer.
+
+We can however, artificially suppress the energy exchange resonances by limiting the number of bosons in the system to 2. We can motivate that this is a good approximation by referring to the Hinton diagram (Fig 3) from the [last tutorial](https://nbviewer.org/github/project-ida/two-state-quantum-systems/blob/master/05-excitation-transfer.ipynb).
```

Comment:
I think it would make more sense to add that to tutorial 4 where we explore down conversion. An additional option it to make some comment near the end when we talk about values of DeltaE?